www/js/diary/list.js

```diff
@@ -726,13 +731,14 @@ angular.module('emission.main.diary.list',['ui-leaflet',
       $scope.draftMode.label = mode.value;
       Logger.log("in storeMode, after setting mode.value = " + mode.value + ", draftMode = " + JSON.stringify($scope.draftMode));
       var tripToUpdate = $scope.modeTripgj;
-      Timeline.instantSave(MODE_CONFIRM_KEY, $scope.draftMode).then(function () {
+      Timeline.instantSave(DESTINATION_CONFIRM_KEY, $scope.draftMode).then(function () {
```

Comment:
like here. You are not saving the `draftMode` to the server, but the `draftDest`. Renaming is a purely syntactic change, not a semantic change, but it makes it a lot easier for others to read and understand.